Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env
+WORKDIR /app
+    
+COPY src/FoodTrans.Auth/*.csproj ./
+RUN dotnet restore
+    
+COPY src/FoodTrans.Auth ./
+RUN dotnet publish -c Release -o out
+    
+FROM mcr.microsoft.com/dotnet/aspnet:6.0
+WORKDIR /app
+COPY --from=build-env /app/out .
+ENTRYPOINT ["dotnet", "FoodTrans.Auth.dll"]
+
```

Comment:
Nie jestem pewien, czy tutaj nie powinieneś dać zmiany użytkownika. Wiesz, chodzi tutaj o bezpieczeństwo aplikacji. Bo default to root.

Reply:
Szczerze, nigdy tego nie robiłem jeszcze 😄 co mam dodać?

Reply:
No na pewno możesz sobie ogarnąć coś takiego

```dockerfile
# create group for /var/www
RUN groupadd -g 1000 www

# Możesz sobie poczytać więcej o tym tutaj -> https://manpages.ubuntu.com/manpages/xenial/man8/groupadd.8.html

# create user for application
RUN useradd -u 1000 -ms /bin/bash -g www www #albo jaki tam katalog chcesz oraz usera

# No i tak naprawdę tutaj -> https://linuxize.com/post/how-to-create-users-in-linux-using-the-useradd-command/

USER www
```

No i przede wszystkim, przyda Ci się możliwość działania w kontenerze. Przyda się też aktualizacja
```Dockerfile
RUN apt update -y && apt upgrade -y
RUN apt install -y build-essential
```
No i co tam później chcesz. Jakiś git, curl, wget czy cokolwiek no 
Upewnij się że masz dobrego EXPOSE'a żeby go złapać w **docker-compose**.

Popraw to proszę sprawdź czy działa i wrzuć zmiankę :D

Reply:
Dodane